modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go

```diff
@@ -102,7 +102,7 @@ func (o *OrgFormModal) SetProps() {
 				"rules": []interface{}{
 					map[string]interface{}{
 						"msg":     "由小写字母、数字，-组成(不允许纯数字)",
-						"pattern": "/^(?![0-9]+$)[a-z0-9-]*$/",
+						"pattern": "/^(?![0-9]+$)^[^-](.*[^-])?$[a-z0-9-]*$/",
```

Comment:
make a comment describe what this regexp means, it's too complex

Reply:
same with orgReg, but Different front-end writing